sklearn/svm/classes.py

```diff
@@ -565,8 +565,7 @@ class SVC(BaseSVC):
         ``1 / (1 + exp(decision_value * probA_ + probB_))``
         where ``probA_`` and ``probB_`` are learned from the dataset. For more
         information on the multiclass case and training procedure see section
-        8 of LIBSVM: A Library for Support Vector Machines (in References)
-        for more.
+        8 of LIBSVM: A Library for Support Vector Machines [1]_.
```

Comment:
I think `[2]_` should also be cited somewhere. Perhaps under the `probability` parameter description above.